language_names/build_db.py

```diff
@@ -0,0 +1,33 @@
+import csv
+import sqlite3
+import sys
+import textwrap
+
+
+def insert_values(c, filename):
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS languageNames (
+            id INTEGER PRIMARY KEY,
+            lg TEXT,
+            inLg TEXT,
+            name TEXT,
+            UNIQUE(lg, inLg) ON CONFLICT REPLACE);
+        """))
+    with open(filename, 'r') as f:
+        reader = csv.DictReader(f, delimiter='\t')
+        for row in reader:
+            c.execute('INSERT INTO languageNames VALUES (?, ?, ?, ?)', (None, row['lg'], row['inLg'], row['name']))
+
+
+def populate_database():
+    conn = sqlite3.connect(sys.argv[-1])
+    c = conn.cursor()
+    c.execute('PRAGMA foreign_keys=OFF;')
+    c.execute('BEGIN TRANSACTION;')
+    [insert_values(c, filename) for filename in sys.argv[1:-1]]
```

Comment:
Please don't do this. Misusing list comprehensions like this is a gross violation of their semantics. Also, your slice can simply be `sys.argv[2:]` (after you switch the order of the arguments.

Reply:
Ohh okay I will follow this.